optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -495,6 +524,35 @@ public Acceptor buildAcceptor(HeuristicConfigPolicy configPolicy) {
                     stepCountingHillClimbingSize_, stepCountingHillClimbingType_);
             acceptorList.add(acceptor);
         }
+        if ((acceptorTypeList!= null && acceptorTypeList.contains(AcceptorType.GREAT_DELUGE))
+                || greatDelugeInitialWaterLevel != null
+                || greatDelugeWaterLevelIncrementScore != null
+                || greatDelugeWaterLevelIncrementRatio != null) {
+            GreatDelugeAcceptor acceptor = new GreatDelugeAcceptor();
+
+            if (greatDelugeWaterLevelIncrementScore != null && greatDelugeWaterLevelIncrementRatio != null) {
+                throw new IllegalArgumentException("The greatDelugeWaterLevelIncrementScore ("
```

Comment:
I don't think the default has been tested. Fixing that now, I 'll submit a separate PR for your review.